### DISCUSSION TITLE
create-next-app --skip-git command

### DISCUSSION THREAD

**Discussion by yoomaxxx** [vercel/next.js#44943] [Ideas]
### Describe the feature you'd like to request

create-next-app --skip-git command

### Describe the solution you'd like

Creating Next.js app without .git folder

### Describe alternatives you've considered

rm -rf .git

**Comment by arvl130** [comment]
I have code changes to get this working already. I was wondering if users or maintainers are interested in getting this feature added in the `create-next-app` cli. Who should we ask?
